src/plugins/plugin.ts

```diff
@@ -181,7 +183,8 @@ export abstract class Plugin<TPluginState extends IPluginState> extends Entity<T
       numberOfBoostedProposals: Number(item.numberOfBoostedProposals),
       numberOfPreBoostedProposals: Number(item.numberOfPreBoostedProposals),
       numberOfQueuedProposals: Number(item.numberOfQueuedProposals),
-      version: item.version
+      version: item.version,
+      isRegistered: item.isRegistered
```

Comment:
why we need to add in 3 different places ?
can it all be consolidate somehow ?

Reply:
As you can see all the other fields are also duplicated probably because a bad design of arc.js from the beginning.

Reply:
Looks like `pluginRegistrar` and `unknownPlugin` are unique in their structure in arc.js. Need to investigate to see if there is a quick fix for that.